apps/vr-tests/src/stories/Tabs.stories.tsx

```diff
@@ -3,7 +3,7 @@ import Screener from 'screener-storybook/src/screener';
 import { storiesOf } from '@storybook/react';
 import { FabricDecorator } from '../utilities/index';
 import { Icon, Fabric } from '@fluentui/react';
-import { Tabs, TabItem, TabItemProps } from '@fluentui/react-tabs/lib/next';
+import { Tabs, TabItem, TabItemProps } from '@fluentui/react-tabs';
```

Comment:
we should have vr test to cover both react-tabs and react-internal/pivots

Reply:
could you also double check screener for other components are pointing to `react` not `react-*`? but no need to duplicate the tests for both since the "next" version is not ready at all to worth the testing

Reply:
There's already a vr-test for `Pivots`, it just didn't need any changes.